application-diagram-ui/src/main/resources/Diagram/DiagramEditSheet.xml

```diff
@@ -650,13 +663,13 @@ define('diagram-store', ['jquery', 'xwiki-meta', 'xwiki-utils', 'diagram-utils',
 
   var convertTextElementToSvg = function(element, s, w, h, htmlConverter, str, rectWidth, isFirstChild) {
     if (element.tagName.toLowerCase() == 'br') {
-      return;
+      return document.createElementNS('http://www.w3.org/2000/svg', "text");
     }
     if (isFirstChild) {
       takenCoordinates = [];
     }
-    w = Math.round(w / 2);
-    h = Math.round((h + s.fontSize) / 2);
+    [w, h] = getAdjustedInitialCoordinates(w, h, s.fontSize);
```

Comment:
Doesn't seem to be supported in IE unfortunately.. see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Browser_compatibility .